.travis.yml

```diff
@@ -2,7 +2,11 @@ language: java
 jdk:
   - openjdk8
   - openjdk11
-
+services:
+  - docker
+before_install:
+  - travis_wait docker-compose -f openmrs-compose.yaml up -d
```

Comment:
Should we do a `curl` too? IIUC the first request to the RefApp UI will trigger some initialization that can take some time as well, right? And if that is the case then I wonder if we need a `travis_wait` for that too. Another preferred solution is to put the `docker-compose` command, the `curl` request after that, and any other initialization step, in a script `foo.sh` and just `travis_wait foo.sh`. That way people can repeat those steps in their local env. easily. WDYT?

Reply:
Yes I think moving these steps that takes long to run to a separate file makes sense so that we can only run ```travis_wait``` once

Reply:
Following up on this thread: Should we also move the `curl` command to the new `travis.sh`?

Reply:
@bashir2 adding this check to `travis.sh` means that both `docker-compose -f openmrs-compose.yaml up -d` and the curl check will be executed in the same lifecycle `before_install`. Having both in the same life cycle the curl command is executed before the container is up hence it fails. It's not possible to use travis lifecycle inside the sh file...

Reply:
I see, okay, never mind then.